packages/kit/src/runtime/server/page/render.js

```diff
@@ -358,15 +367,29 @@ export async function render_response({
 				hydrate.push(`params: ${devalue.uneval(event.params)}`, `route: ${s(event.route)}`);
 			}
 
-			args.push(`{\n\t\t\t\t\t\t\t${hydrate.join(',\n\t\t\t\t\t\t\t')}\n\t\t\t\t\t\t}`);
+			const indent = load_env_eagerly ? '\t\t\t\t\t\t\t' : '\t\t\t\t\t\t';
```

Comment:
This is super nitpicky but is there a reason we can't just do `'\'.repeat(x)`? I manually counted these to make sure we weren't just doing the same number in both cases 😆

Reply:
```suggestion
			const indent = '\t'.repeat(load_env_eagerly ? 7 : 6);
```